contrib/scripts/ha/ha_make_slavedb.sh

```diff
@@ -16,7 +16,7 @@ repl_log_home=$CUBRID_DATABASES
 db_name=
 
 backup_dest_path=
-backup_option=
+backup_option="-z --no-check"
```

Comment:
1. These two options seem to be always set if no options are given.
2.  Given any option, these seem to be overwritten.

Am I correct and is it correct?

Reply:
The parameters in this "User Configuration" part are set manually by modifying the script if necessary. 
So, I wonder why the backup option needs this initial value ?

Reply:
Before each replies is tagged, there is the notice.
Tech team requested that two options,-z and --no-check, are written explicitly.

Reply:
@wrlawodms 

1. --no-check option is set to default and -z option is set to default.

> --no-check               don't check a consistency of the database; default: check

2. Can I ask the meaning "overwritten"? does it stands for options written more than once?
Are there any problems if they are written more than once?

Reply:
@joohok 
Did my first reply help your reply? Or are there any something what I missed?

Reply:
@eido5 What do you mean by "explicitly"? `z` and `--no-check` is special ones than other options?
Other options have to be written into `backup_option` to be used with `cubrid backupdb` below, but why are the two options set by default?

Reply:
Furthermore, `-z` option is even not printed in the help message of `cubrid backupdb` and compression is applied in default now.

Reply:
> Furthermore, `-z` option is even not printed in the help message of `cubrid backupdb` and compression is applied by default now.

Right. I should write down about the -z option with no.2 . 
[CBRD-24032](http://jira.cubrid.org/browse/CBRD-24032)

I can guess from your reply that "overwritten" seems no problem.
Tech team requested the addition options explicitly for the compatibility with the previous CUBRID version.

Reply:
Regrading "overwritten", I guess that the variables can be set out of the script, but I'm not sure it's possible. If it is impossible, never mind. My opinion is basically the same as @joohok's.

Reply:
The request is that '-z' and '--no-check' have to be always set when no option is given, and if the option is given(means `backup_option` is changed) it doesn't need to be set?

Reply:
> To clarify, the request is that '-z' and '--no-check' have to be always set when no option is given, and if the option is given(means `backup_option` is changed) it doesn't need to be set?

I understood your reply that '-z' and '--no-check' is always set whether `backup_option` is changed.
If it, yes it is. 

the request includes that it is necessary for the compatibility with any version CUBRID. 
So `backup_option` is set to  '-z' and '--no-check' options.

Can i ask the case of `backup_option` is given?(I may guess it is transfer by any argument.)

Reply:
> `Can i ask the case of backup_option is given?(I may guess it is transfer by any argument.)`

I thought something like `backup_option='[other option]' ./ha_make_slavedb.sh `, but it will be overitten by `backup_option="-z --no-check"` here. Never mind.

> If it, yes it is.

1. When a user changes the `backup-option` , the -z and --no-check can be removed. Is it part of the request?
2. In other cubrid versions, `--no-check` and '-z' is the default behavior of backupdb? There are -z and --no-check in 10.1 too. What do you mean by "the compatibility"?

Reply:
> 1. When a user changes the `backup-option` , the -z and --no-check can be removed. Is it part of the request?
 Thank you. I didn't predict this problem. I should postpone to reply this question until I receive the answers from the tech team.
 
> 2. In other cubrid versions, `--no-check` and '-z' is the default behavior of backupdb? There are -z and --no-check in 10.1 too. What do you mean by "the compatibility"?
I used "the compatibility" for the previous and next version cubrid. 
I know that after this issue, [CBRD-24032](http://jira.cubrid.org/browse/CBRD-24032), compress '-z' option is set to default.
So, I think these options can be set to default or not in any versions.

Reply:
I understand what you mean. 
Before requesting review, please explain why this work is required in the jira. 
I thought this default option was necessary to set the minimum options the script would run on, and I thought there is no problem to execute this script without that backup options you describe. 
So if the user wants the '-z' and '--no-check' options, the user can change the options themselves. But with the "compatibility" view, I think it is OK. 
 is '--no-check' option also added for the compatibility issue?

Reply:
@joohok 
> Before requesting review, please explain why this work is required in the jira. 

Oh I see. My one is lack of explanation. 

> Is '--no-check' option also added for the compatibility issue?

In my opinion, the compatibility includes '--no-check' option. It is related to the back-up performance. 
So Tech team needs to write on scripts even '--no-check' option.

Is '--no-check' option not enough relation with 'compatibility'?